### ISSUE TITLE
Can't enable

### ISSUE THREAD

**Issue by kevinwahyudi** [alterstep/dnscrypt-osxclient#117]
DNScrypt not enabled even after enabling it.
installed using GUI osx installer dmg file, on OSX 10.12

![screen shot 2016-10-26 at 1 16 46 pm](https://cloud.githubusercontent.com/assets/2149365/19715475/9ecf6f6a-9b7f-11e6-9b3b-01625f241f53.png)

**Comment by sgama** [comment]
+1

**Comment by sorcer1122** [comment]
I have suddenly got myself into the same situation. It was perfectly working but then one day it stopped working. Any fix?

**Comment by rahadiancs** [comment]
happened to me today. it was working before. tried to reinstall, still not fixed.

**Comment by sorcer1122** [comment]
@rahadiancs THANK YOU! I had a VPN client running and after I closed it DNScrypt started to work again!

**Comment by DerekCurrie** [comment]
Same situation. I've uninstalled and reinstalled many times, including the Menubar.app. I checked my VPN's configuration for a conflict. I've quit my VPN. Did Not Help.

Oddity #1: I noticed is that the DNSCrypt System Preferences Pane never ever keeps any setting I place in it. Examples: If I change anything in the Advanced tab, it never sticks. I can no longer 'Log DNS queries'. I can no longer '...use the following static DNS servers:'. The same goes with the General tab. Nothing sticks. Could this be a permissions problem?

Oddity #2: I'm using the final version of 10.12.2 on two Macs. I'm having the problem on my MacBook Pro late 2013. But I am NOT having the problem on my Mac Mini 2011. It's working perfectly there. AND I have the very same VPN (proXPN) installed and running (but not connected) on BOTH machines.

Oddity #3: After uninstalls & reinstalls, DNSCrypt makes no effort at all to set up its required DNS settings, as found in the Network System Preferences pane. It doesn't bother any more. Everything remains the same. What changed?

Apart from just using the uninstall script inside the About tab, I also fished around for every other file related to DNSCrypt and deleted that as sell, including both the Menubar.app, its plist file and its Application Support folder. Have I missed something obscurely named that could have become corrupt?

In the meantime, I either use secure DNS provided inside my VPN or I use OpenDNS. (And no, turning off secure DNS in my VPN doesn't solve the problem).
